src/encoded/schemas/hic_quality_metric.json

```diff
@@ -0,0 +1,349 @@
+            "minimum": 0,
+            "maximum": 100.0
+        },
+        "unique_reads": {
+            "title": "Number of unique reads",
+            "type": "integer",
+            "minimum": 0
+        },
+        "pct_unique_reads": {
+            "title": "Percentage of unique reads",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
+        "duplicates": {
+            "title": "Number of duplicates",
+            "type": "integer",
+            "minimum": 0
+        },
+        "pct_sequenced_duplicates": {
+            "title": "Percentage of sequenced duplicates",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
+        "pct_alignable_duplicates": {
+            "title": "Percentage of alignable duplicates",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
+        "library_complexity_estimate": {
+            "title": "Number of duplicates",
+            "type": "integer",
+            "minimum": 0
+        },
+        "intra_fragment_reads": {
+            "title": "Number of intra fragment reads",
+            "type": "integer",
+            "minimum": 0
+        },
+        "pct_sequenced_intra_fragment_reads": {
+            "title": "Percentage of sequenced intra fragment reads",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
+        "pct_unique_intra_fragment_reads": {
+            "title": "Percentage of unique intra fragment reads",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
+        "below_mapq_threshold": {
+            "title": "Number of reads below mapq threshold",
+            "type": "integer",
+            "minimum": 0
+        },
+        "pct_sequenced_below_mapq_threshold": {
+            "title": "Percentage of sequenced reads below mapq threshold",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
+        "pct_unique_below_mapq_threshold": {
+            "title": "Percentage of unique reads below mapq threshold",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
+        "hic_contacts": {
+            "title": "Number of hic contacts",
+            "type": "integer",
+            "minimum": 0
+        },
+        "pct_sequenced_hic_contacts": {
+            "title": "Percentage of sequenced reads in hic contacts",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
+        "pct_unique_hic_contacts": {
+            "title": "Percentage of unique reads in hic contacts",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 100.0
+        },
+
+        "pct_3_prime_bias_long_range_min": {
+            "title": "Minimum of the 3' bias long rage percentage",
```

Comment:
typo, should be `range` instead of `rage`

Reply:
fixed